b64json/b64json.go

```diff
@@ -3,8 +3,37 @@ package b64json
 import (
 	"encoding/base64"
 	"encoding/json"
+	"github.com/iancoleman/orderedmap"
 )
 
+func indexOf(needle string, data []string) int {
+	for k, v := range data {
+		if needle == v {
+			return k
+		}
+	}
+	return -1
+}
+
+func jsonSort(a *orderedmap.Pair, b *orderedmap.Pair) bool {
+	jsonOrder := []string{
+		"trustchain_id",
+		"target",
+		"value",
+		"delegation_signature",
+		"ephemeral_public_signature_key",
+		"ephemeral_private_signature_key",
+		"user_secret",
+		"public_encryption_key",
+		"private_encryption_key",
+		"public_signature_key",
+		"private_signature_key",
+	}
+	aIdx := indexOf(a.Key(), jsonOrder)
+	bIdx := indexOf(b.Key(), jsonOrder)
```

Comment:
Why not a map like you did in other languages?

Reply:
Good point, fixed.